services/src/main/java/org/exoplatform/addons/gamification/listener/social/activity/GamificationActivityListener.java

```diff
@@ -292,6 +292,9 @@ private boolean isSpaceActivity(ExoSocialActivity activity) {
   }
 
   private boolean isDocumentShareActivity(ExoSocialActivity activity) {
+    if (activity.getType() == null) {
+      return false;
+    }
     return activity.getType().equalsIgnoreCase("files:spaces") || activity.getType().equalsIgnoreCase("DOC_ACTIVITY")
         || activity.getType().equalsIgnoreCase("contents:spaces");
```

Comment:
Can you make all the test in the return ? without a if

Reply:
done